pkg/api/conversion.go

```diff
@@ -28,12 +30,48 @@ import (
 var Codec = runtime.CodecFor(Scheme, "")
 
 func init() {
+	Scheme.AddDefaultingFuncs(
```

Comment:
Smart defaulting for ListOptions deserialized from an incoming request.  Rather than make all people using it set those to empty and risk doing something wrong.

----- Original Message -----
> > @@ -28,12 +30,48 @@ import (
> >  var Codec = runtime.CodecFor(Scheme, "")
> >  
> >  func init() {
> > +	Scheme.AddDefaultingFuncs(
> 
> Why is this necessary? I'd like to nuke this whole file.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/5763/files#r27089694